.gitignore

```diff
@@ -0,0 +1,4 @@
+_fresh/
+deno_modules/
```

Comment:
Not sure why you include this, given it was only named that for (less than(?)) one unstable version? I would remove this to avoid any confusion.